database/migrations/2024_02_01_000000_add_permissions_for_manual_trips_beta.php

```diff
@@ -1,25 +1,11 @@
 <?php
 
 use Illuminate\Database\Migrations\Migration;
-use Spatie\Permission\Models\Permission;
-use Spatie\Permission\Models\Role;
 
 return new class extends Migration
 {
 
     public function up(): void {
-        $permission = Permission::create(['name' => 'create-manual-trip']);
-
-        Role::findByName('admin')->givePermissionTo($permission);
-        Role::findByName('closed-beta')->givePermissionTo($permission);
-    }
-
-    public function down(): void {
-        $permission = Permission::findByName('create-manual-trip');
-
-        Role::findByName('admin')->revokePermissionTo($permission);
-        Role::findByName('closed-beta')->revokePermissionTo($permission);
-
-        $permission->delete();
+        //empty now - not deleted so a revert doesn't break
```

Comment:
<img width="1194" alt="image" src="https://github.com/Traewelling/traewelling/assets/1267894/b5efea69-8e74-4cf5-8ac9-979fb8ec858c">

You can apparently delete migrations without any problems. Have you tested what happens in case of a rollback with these specific migrations?